design-library/source/_patterns/01-atoms/09-social-icon/_social-icon.scss

```diff
@@ -1,17 +1,28 @@
-li.social_media--item {
+.social-icon {
   display: inline-block;
-  padding: 0;
-  margin: 0;
   height: 2em;
+  margin: 0 auto;
+  padding: 0;
   width: 2em;
 
-}
+  a {
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: block;
+    height: inherit;
+    text-indent: -9999px;
+    width: inherit;
+
+    &.Twitter {
```

Comment:
we may want to make/move this to content when we integrate into the CMS

Reply:
We are not using this, we are using the images as background images. I can change this to work like the icons we are using for tags.